### PR TITLE
New package: ocean-sound-theme-6.1.4

### PR DESCRIPTION
**Pull request by hunt3r0one**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

This is the default sound theme for KDE Plasma 6.

### CODE REVIEW

File: srcpkgs/ocean-sound-theme/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+version=6.1.4
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules qt6-tools qt6-base"
+makedepends="qt6-tools-devel"
+short_desc="Ocean Sound Theme for Plasma"
+maintainer="Hunter Ellett <hunter2k1@disroot.org>"
+license="custom:multiple"
+homepage="https://invent.kde.org/plasma/ocean-sound-theme"
+distfiles="https://invent.kde.org/plasma/${pkgname}/-/archive/v${version}/${pkgname}-v${version}.tar.gz"
```
Comment (Johnnynator) [inline srcpkgs/ocean-sound-theme/template:12]:
```suggestion
distfiles="${KDE_SITE}/plasma/${version}/${pkgname}-${version}.tar.xz"
```

File: srcpkgs/ocean-sound-theme/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'sound-theme-ocean'
+pkgname=ocean-sound-theme
+version=6.1.4
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules qt6-tools qt6-base"
+makedepends="qt6-tools-devel"
+short_desc="Ocean Sound Theme for Plasma"
+maintainer="Hunter Ellett <hunter2k1@disroot.org>"
+license="custom:multiple"
```
Comment (Johnnynator) [inline srcpkgs/ocean-sound-theme/template:10]:
```suggestion
license="CC-BY-SA-4.0"
```

(CI stuff is `CC0-1.0` but not used from us, CMake stuff is `BSD-2-Clause` but not installed, so not really important for the final package)

**Review by Johnnynator — CHANGES_REQUESTED** [review]

**Comment by ahesford** [comment]
The commit message should be

    New package: ocean-sound-theme-6.1.4

**Comment by hunt3r0one** [comment]
How come it was never accepted?

**Pull request closed without merging**